### PR TITLE
Optimise favicon, social image

### PR DESCRIPTION
**Pull request by fairlighteth**
- Optimizes favicon.png size
- Removes duplicate favicon references
- Remove duplicate meta viewport
- Adds og:meta social image asset
og-meta-cowprotocol

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -39,29 +39,28 @@ export default function App(props: AppProps) {
...
         <meta name="description" content={siteConfig.description} />
-        <meta name="theme-color" media="(prefers-color-scheme: light)" content="white" />
-        <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#163861" />
+        <meta name="theme-color" media="(prefers-color-scheme: light)" content="black" />
+        <meta name="theme-color" media="(prefers-color-scheme: dark)" content="black" />
 
         <link rel="shortcut icon" type="image/png" href="/favicon.png" />
-        <link rel="apple-touch-icon" sizes="192x192" href="/images/logo-square-192.png" />
-        <link rel="apple-touch-icon" sizes="512x512" href="/images/logo-square-512.png" />
+        <link rel="apple-touch-icon" sizes="192x192" href="/favicon.png" />
```
Comment (anxolin) [inline pages/_app.tsx:47]: Is it ok to use the 512x512 one here?

**Review by anxolin — APPROVED** [review]
Awesome

**Comment by elena-zh** [comment]
LGTM

File: pages/_app.tsx (hunk 1 above)
Comment (fairlighteth) [reply pages/_app.tsx:47]: For simplicity now I'd say yes. But for future iterations we could use a guide like this one https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs (tl:dr; mostly offsetting to a webmanifest file and using specific sizes)

**Pull request merged**